src/components/RadioGroup/RadioGroup.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { VFC, memo, ReactNode, useCallback } from 'react';
+import { FormControl, FormErrorMessage, RadioGroup as ChakraRadioGroup, VStack } from '@chakra-ui/react';
+import { useController } from 'react-hook-form';
+import { FormControlProps } from '../../utils/formControl';
+
+interface Props extends FormControlProps {
+
+  /** Radio button props. */
+  readonly children: ReactNode;
```

Comment:
As I see, you expect from props radio buttons based on docs, but here you actually have `ReactNode` type for `children` which means that you can pass in your component literally anything. Can we improve that?

Reply:
I agree with doc, but `chakra-ui` uses the same approach with `children`. We can specify type for direct children, but in order to save ability to customize the component we use children as required prop.